employee/members/views.py

```diff
@@ -0,0 +1,88 @@
+from django.shortcuts import render
+
+# Create your views here.
+from django.http import HttpResponse
+from django.template import loader  
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+import json
+#from .serializers import EmployeeSerializer
+
+
+def index(request):
+  template = loader.get_template('first.html')
+  return HttpResponse(template.render())
+  
+@method_decorator(csrf_exempt, name='dispatch')
+class EmployeeView(View):
+     def get(self, request, employee_id=None):
+          employee_model_list = []
+          try:
+             if employee_id:
+                employee_model_list = Employees.objects.filter(employee_id=employee_id)
+        
+          except Employees.DoesNotExist:
+             return JsonResponse({'status': 'failed', "employees": None}, status=400)
```

Comment:
@AnkitaHegde Please make sure that the response sent out for failure indicates the reason for failure.